academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md

```diff
@@ -2,24 +2,30 @@
 ms.topic: reference
-ms.date: 2020-02-10
+ms.date: 2020-04-15
 ---
 
 # Troubleshoot Common MAKES command line tool issues
 
 ## Azure login failure due to multiple subscriptions or multiple tenants being tied to a single Azure account
 
-If you have multiple azure tenants associated with a single azure account, you'll need to specify the Azure Active Directory directory domain name associated with the Azure Subscription that you would like to use to execute the command. (e.g. "--AzureActiveDirectoryDomainName constco.onmicrosoft.com").  
+If you have multiple Azure tenants associated with a single Azure account, you'll need to specify the Azure Active Directory directory domain name associated with the Azure Subscription that you would like to use to execute the command. (e.g. "--AzureActiveDirectoryDomainName constco.onmicrosoft.com").  
 
-You can find this information by logging into azure portal, searching for "Azure Active Directory" and viewing resource detail page. The domain name to use will be in the "Overview" section at the top left.
+You can find this information by logging into Azure Portal, searching for "Azure Active Directory" and viewing resource detail page. The domain name to use will be in the "Overview" section at the top left.
 
 If you have multiple subscriptions associated with the same Azure Active Directory doamin and want to use the non-default subscription, you'll need to specify the subscription Id associated with the subscription to execute the command. (e.g. "--AzureSubscriptionId XXXXXXX-XXXX-XXX-XXXX-XXXXXXX")
 
-You can find this information by logging into azure portal, searching for "Subscriptions" and viewing resource detail page. The subscription Id will be listed in the table.
+You can find this information by logging into Azure Portal, searching for **Subscriptions** and viewing resource detail page. The subscription Id will be listed in the table.
 
-## Azure subscription limit
+## Azure Subscription quota limit reached
 
-If you encounter errors such as "operation could not be completed as it results in exceeding approved XXXXXX quota", you'll either need to contact azure support to increase your account quota or change the MAKES host machine settings.  
+If you encounter errors such as "operation could not be completed as it results in exceeding approved XXXXXX quota", you'll need to contact Azure support to increase your account quota, try a different region where you have more Azure quota, or change the MAKES host machine sku/worker machine sku settings. For most Microsoft Academic sub-graph index builds, you can scale down to **Standard_D8_v3** sku to build and host the indexes.
+
+## Azure Batch service not registered
+
+If you encouter errors such as "The subscription registration is in 'Unregistered' state. The subscription must be registered to use namespace 'Microsoft.Batch'....", you'll need to register your subscription to use Azure Batch service.
```

Comment:
"If you encouter errors such as "The subscription registration is in 'Unregistered' state. The subscription must be registered to use **the** namespace 'Microsoft.Batch'..."

Reply:
copied error message word for wrod.